### DISCUSSION TITLE
Lower PV power output at location with higher GHI

### DISCUSSION THREAD

**Discussion by power-hunter** [pvlib/pvlib-python#1897] [Q&A]
I am trying to do some very simple calculation of PV power output at two locations: location A (lat: 42.142, lon: 109.252) and B (lat: 46.152, lon: 122.424).  From PVGIS data, location A is with higher GHI however PVLIB gives a lower PV power output.  My pvlib model is just a minor modification on the example given in the ModelChain example in the user guide.  I'm wondering if some one can give me some hint on the result?  Thank you!

```python
import pvlib
import pandas as pd
import matplotlib.pyplot as plt
import math

# latitude, longitude, name, altitude, timezone
coordinates = [
    (42.142, 109.252, 'Damao', 1277, 'Etc/GMT-1'),
#    (46.152, 122.424, 'Hinggan', 200, 'Etc/GMT-8')
]
    
# get the module and inverter specifications from SAM
sandia_modules = pvlib.pvsystem.retrieve_sam('SandiaMod')
cec_modules = pvlib.pvsystem.retrieve_sam('CECMod')
sapm_inverters = pvlib.pvsystem.retrieve_sam('cecinverter')
#module = cec_modules[panel_py]
module = sandia_modules['Canadian_Solar_CS5P_220M___2009_']

#inverter = sapm_inverters[inverter_py]
inverter = sapm_inverters['ABB__MICRO_0_25_I_OUTD_US_208__208V_']
temperature_model_parameters = pvlib.temperature.TEMPERATURE_MODEL_PARAMETERS['sapm']['open_rack_glass_glass']


#number_panel = math.floor(inverter['Pdco']/module['STC'])
#pvcap_readin = number_panel*module['STC']
#print(pvcap_readin)


tmys = []

for location in coordinates:
    latitude, longitude, name, altitude, timezone = location
    weather = pvlib.iotools.get_pvgis_tmy(latitude, longitude,
                                          map_variables=True,
                                          url='https://re.jrc.ec.europa.eu/api/v5_2/')[0]
    weather.index.name = "utc_time"
    tmys.append(weather)

from pvlib.pvsystem import PVSystem, Array, FixedMount
from pvlib.location import Location
from pvlib.modelchain import ModelChain

energies = {}

for location, weather in zip(coordinates, tmys):
    latitude, longitude, name, altitude, timezone = location
    location = Location(
        latitude,
        longitude,
        name=name,
        altitude=altitude,
        tz=timezone,
    )
    mount = FixedMount(surface_tilt=latitude, surface_azimuth=180)
    array = Array(
        mount=mount,
        module_parameters=module,
        temperature_model_parameters=temperature_model_parameters,
        modules_per_string = 1,
    )
    system = PVSystem(arrays=[array], inverter_parameters=inverter)
    mc = ModelChain(system, location, aoi_model='physical', spectral_model='no_loss')
    mc.run_model(weather)
    #annual_energy = mc.results.ac.sum()
    ts_df = mc.results.ac.reset_index()
    ts_df.rename(columns = {0:'data'}, inplace = True)

print("Executed python_script.py successfully!")
print(sum(ts_df['data'])/220)
```

**Comment by mikofski** [comment]
1. PVGIS always returns times in UTC, so make sure this isn’t a time zone issue.

2. Compare the annual plane of array irradiance (POA) sometimes also called global tilted incident irradiance (GTI). Diurnal variations in diffuse light can mean that even though GHI is larger, the actual incident irradiance can be less

**Reply by power-hunter** [reply]
Hi Mikofski,

Thank you for the reply.  Seems the issue is not from time zone after checking.  I noticed for this two particular locations, PVGIS are using different default irradiance databases, one with SARAH and the other with ERA5.  For location 42.142, 109.252 irradiance from both databases are available and they are quite different from each other.  Perhaps this is why I am seeing the discrepancy?

**Reply by mikofski** [reply]
check plane of array irradiance, should be in `mc.results.total_irrad['poa_global']` you can sum it up and it should track with output, EG: larger POA -> larger energy output. However, POA may differ from GHI because of diurnal variations in diffuse light. EG: if it's cloudy in the middle of the day at one site and only clear in the afternoon, then even if it's GHI is larger, its POA may be lower, because there's less DNI midday when it has the most impact. You can view a few days during each season to compare. Try `mc.results.total_irrad[start_date:end_date].plot()` where `start_date` and `end_date` are the timestamps of the days you want to plot
